pages/common/pbmtoascii.md

```diff
@@ -0,0 +1,20 @@
+# pbmtoascii
+
+> Convert a PBM image to ASCII graphics.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtoascii.html>.
+
+- Read PBM file as input and produce an ASCII output:
+
+`pbmtoascii {{path/to/input_file}}`
+
+- Read PBM file as input and save an ASCII output into a file:
+
+`pbmtoascii {{path/to/input_file}} > {{path/to/output_file}}`
+
+- Read PBM file as input setting the pixel mapping (defaults to 1x2):
+
+`pbmtoascii -2x4 {{path/to/input_file}}`
```

Comment:
```suggestion
- Read a PBM file as input while setting the pixel mapping (defaults to 1x2):

`pbmtoascii -{{2x4}} {{path/to/input_file}}`
```

Reply:
In this case, the only possible values are `1x2` and `2x4`. Do you think we need a new syntax in that case?